docs/getting_started/index.md

```diff
@@ -227,32 +227,39 @@ If you grab the placeholder sphere icon, you'll see your gadget's panel.
 ![Gadget panel](gadget_panel.jpg "Gadget Panel")
 
 
-# Step 4 - Understanding gadget_manifest.json
+# Step 4 - Understanding manifest.webmanifest
 
-Your gadget's src directory contains a file called gadget_manifest.json that has the following in it.
-```json
+Your gadget's src directory contains a file called manifest.webmanifest that has the following in it.
+```webmanifest
 {
+	"xr_type": "aardvark-gadget@^0.6.6",
 	"name": "My Awesome Gadget",
-	"permissions": [
-		"scenegraph"
+	"icons": [
+		{
+			"src": "models/placeholder.glb",
+			"type": "model/gltf-binary"
+		}
 	],
-	"width": 1024,
-	"height": 1024,
-	"model": "models/placeholder.glb",
-	"startAutomatically": false
+	"aardvark": {
+		"permissions": [
+			"scenegraph"
+		],
+		"browserWidth": 1024,
+		"browserHeight": 1024,
+		"startAutomatically": false
+	}
 }
 ```
 
 Every gadget needs to define a manifest file to tell Aardvark how to deal with that gadget. The fields in the manifest file are:
 * name - The user facing name of the gadget. This isn't currently shown to a user anywhere, but please set one anyway.
+* icons - The URL of a glTF model to use to represent this gadget in the gadget menu. If this does not start with HTTP or HTTPS it must be a relative file path (using forward slashes) to the model file that is relative to the gadget manifest file itself. **(note that if you use a custom icon you currently need to ensure it's placed in `dist/models`)** 
```

Comment:
custom models should work from anywhere inside the gadget, including the root of `dist`. Are they not?

Reply:
yeah this can probably be better phrased. 
I want to indicate that models are _not_ referenced from the `src` folder despite the fact we're editing the manifest in that folder. To someone assuming a "work in src, then webpack transpiles and bundles all the things to `dist`, + copies over necessary assets" flow (as happens w/image assets going through compression for example) this might be unintuitive. (was for me, in any case.) And placeholder.glb _is_ placed in `dist` upon building which adds to the ambiguity. 

How about something like "the relative path to the model should be from the `dist` folder, since that's where the manifest file will be"?

Reply:
That makes sense, but aren't the relative paths the same in both cases? Manifest in ./src/manifest.webmanifest and source models in ./src/models. Then manifest in ./dist/manifest.webmanifest and output models in ./dist/models

Reply:
yes, but the question is where would you put the actual model. since all the action is in `src`, and placeholder.glb exists in `src/models` as well (needlessly?), my natural thinking would be that all models go in `src`.

Reply:
(note removed based on our other discussion.)